content/documentation/nodes/action/_index.md

```diff
@@ -1,8 +1,18 @@
 ---
 title : Action
+weight : 16
 chapter : true
 ---
 
 # Action
 
-This is the documentation for actions.
+### Data Types
+
+- **Action** - A data structure that represents an action. An action is an abstract representation of an animation of a certain attribute of an object. Actions can be constructed from Blender F-Curves or can be procedurally defined.
+
+### Sockets
+
+- **Action Socket** - It can intake or output the action(s).
+
+### Nodes
```

Comment:
Make sure to have a new line after headlines. Some markdown implementations care about this stuff.

Reply:
Thanks for the info. It is added.